src/databricks/labs/ucx/azure/resources.py

```diff
@@ -117,44 +121,134 @@ def inner() -> dict[str, str]:
 
         return _credentials
 
+    def get(self, path: str, host: str, api_version: str | None = None):
```

Comment:
This is incorrect refactoring.

Concatenate host and path to "url", having two parameters like that is going to lead to errors.